api/jobs/handlers.py

```diff
@@ -335,7 +335,7 @@ def get_config(self, _id):
         # Detect if config is old- or new-style.
         # TODO: remove this logic with a DB upgrade, ref database.py's reserved upgrade section.
 
-        if c.get('config') is not None and c.get('inputs') is not None:
+        if 'config' in c and c.get('inputs') is not None:
```

Comment:
Can you explain the rationale behind this change? This would fail if `c['config']` is None.

Reply:
It was failing when config existed but was not set. AKA, it was the "new" style, but there was no config for the job.